### PR TITLE
KAN-27: fix the double click on conditions form to choose from dropdown

### PR DESCRIPTION
**Pull request by FireEagle97**

### CODE REVIEW

File: components/autocomplete.js [hunk 1]
```diff
@@ -17,14 +17,14 @@ import { reactIfView } from "../global/global-functions";
 const Item = ({item, onPress}) => (
   <TouchableOpacity
   style={[styles.itemButton, flex_style.flex, ,flex_style.alignCenter]}
-  onPress={(event) => onPress(item)}
+  onPress={(event) =>onPress(item)}
```
Comment (kalo2711) [inline components/autocomplete.js:20]: Why is there a duplicate?

File: components/autocomplete.js [hunk 2]
```diff
@@ -33,19 +33,18 @@ const DropdownWithModal = ({ dataset, onChangeText, placeholder, setSelectedItem
...
   useEffect(() => {
     setItems([...dataset]);
-      return () => {
-        keyboardDidHideListener.remove();
-      };
   }, [dataset]);
 
+  useEffect(() => {
+    if (!isModalVisible) {
+      setTimeout(() => Keyboard.dismiss(), 50);
```
Comment (kalo2711) [inline components/autocomplete.js:40]: You are still using a Timeout...

File: global/supportedLanguages.json [hunk 3]
```diff
@@ -90,8 +90,7 @@
     "store": "Store",
     "submit": "Submit",
     "sun": "Sun",
-    "temperature": "Temperature (°C)",
-    "dateExceeds8Days": "*Cannot get weather for dates more that 8 days from now*",
```
Comment (kalo2711) [inline global/supportedLanguages.json:94]: These changes shouldn't be removed, like we talked.

File: global/supportedLanguages.json [hunk 4]
```diff
@@ -248,8 +247,7 @@
     "store": "Magasin",
     "submit": "Soumettre",
     "sun": "Soleil",
-    "temperature": "Température (°C)",
-    "dateExceeds8Days": "*Impossible d'obtenir la météo pour les dates à plus de 8 jours*",
```
Comment (kalo2711) [inline global/supportedLanguages.json:252]: Shouldn't be removed.

File: screens/ConditionsForm.js
Comment (kalo2711) [inline screens/ConditionsForm.js:1]: This file shouldn't be changed. Only your additions need to be there.

File: components/autocomplete.js (hunk 1 above)
Comment (FireEagle97) [reply components/autocomplete.js:20]: delete it

File: components/autocomplete.js (hunk 2 above)
Comment (FireEagle97) [reply components/autocomplete.js]: removed

File: global/supportedLanguages.json (hunk 3 above)
Comment (FireEagle97) [reply global/supportedLanguages.json:94]: I can still see these lines in this file

File: global/supportedLanguages.json (hunk 4 above)
Comment (FireEagle97) [reply global/supportedLanguages.json:252]: I can still see these lines in this file

File: screens/ConditionsForm.js
Comment (FireEagle97) [reply screens/ConditionsForm.js:1]: it's not coming from my end. how can we resolve that?

File: components/autocomplete.js (hunk 2 above)
Comment (stefanoScalzo) [inline components/autocomplete.js:40]: There is still a timeout here...why ?

Comment (FireEagle97) [reply components/autocomplete.js]: now its removed

**Pull request merged**